charts/trusted-attestation-controller/templates/configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+data:
+  KEY_SERVER: localhost:443
```

Comment:
@luukasmakila What is the plan for dealing with more plugin-specific configurations? For example, the iSecL plugin has more config options than just server URL: https://github.com/intel/trusted-attestation-controller/pull/13/files#diff-b6ac01d0d33b89d5494244a80fa7a13c5b4fcb4b3734936bb11138dbafd2739d

Reply:
@avalluri Now it works with the iSecL confirm-a swell. Just copy paste the configuration stuff into values.yaml -> plugin.configYaml.

Reply:
@avalluri any updates on this?